modules/user-guide/pages/linting-vale-cli.adoc

```diff
@@ -0,0 +1,103 @@
+// Metadata for Antora
+:navtitle: Linting with Vale from the command line
+:keywords: vale
+:description: Describes how to run Vale from the command line to check your content for style errors, warnings, and suggestions.
+:page-aliases: end-user-guide:running-vale-cli.adoc
+// End of metadata for Antora
+// Metadata for Modular Docs
+:context: assembly_getting-started-with-vale
+:_module-type: PROCEDURE
+// End of metadata for Modular Docs
+[id="proc_linting-vale-cli_{context}"]
+= Linting your content by running Vale from the command line
+
+To analyze your content to see how it aligns with the link:https://redhat-documentation.github.io/supplementary-style-guide/[Red Hat supplementary style guide], you can run Vale from the command line. You can run Vale on individual files or an entire directory.
+
+.Prerequisites
+
+Ensure that you have completed the following prerequisite steps:
+
+* Installed the latest release link:https://vale.sh/docs/vale-cli/installation/[Vale] and link:https://docs.asciidoctor.org/asciidoctor/latest/[Asciidoctor].
++
+[IMPORTANT]
+====
+To run Vale on AsciiDoc files, Asciidoctor must be installed.
+====
++
+* Configured the `.vale.ini` file of your local Vale client to use the vale-at-red-hat rules, as outlined in xref:installing-vale-cli.adoc[].
+* Pulled the latest link:https://github.com/redhat-documentation/vale-at-red-hat[vale-at-red-hat] style rules from the main repository by running the `vale sync` command or by using another `git pull` sync method.
+
+.Procedure
+
+. Start a terminal session and go to the content folder containing the files that you want to analyze.
+. Enter the `vale` linter command by using the following syntax:
+[source,commandline]
```

Comment:
```suggestion
[source,terminal]
```
I'm not familiar with this source format. I've usually seen `[source,terminal]` for commands like this, and `[source,bash]` for Bash scripts.